x-pack/plugins/apm/server/routes/environments/get_environments.ts

```diff
@@ -39,44 +39,29 @@ export async function getEnvironments({
     ? 'get_environments_for_service'
     : 'get_environments';
 
-  const params = {
+  const params: APMEventESTermsEnumRequest = {
```

Comment:
With the comment [below ](https://github.com/elastic/kibana/pull/150175/files#r1094610101), you can move this to inside `apmEventClient.termsEnum(operationName, params);`

Reply:
I removed this due to the above comment